### ISSUE TITLE
[Dialogs] The Dialog title should have the trait UIAccessibilityTraitHeader

### ISSUE THREAD

**Issue by danblakemore** [material-components/material-components-ios#2916]
### Overview
Dialog titles do not have the header trait and do not announce that they are headers in VoiceOver.

Internal issue b/73126907

### Reproduction steps
 - Display a dialog with a title.
 - Enable VoiceOver
 - Select the title element.

Expected result
 VoiceOver reads "[TITLETEXT], header"
Actual result
 VoiceOver reads "[TITLETEXT]"

### Version number
 - 44.6.0

### Operating system and device
- iOS 11, iPhone X

---
#### Internal data
- Associated internal bug: [b/117179089](http://b/117179089)

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
